### PR TITLE
ahmed/DAPI-618/feat--update-navigation-bar

### PR DESCRIPTION
**Pull request by ahmed-deriv**
Update navigation bar for desktop and mobile

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -353,27 +355,97 @@ div[class*='sidebarViewport'] {
...
+    font-size: rem(1.4);
+  }
+
+  .external-nav-link {
+    &::after {
+      content: '';
+      display: inline-block;
+      position: relative;
+      bottom: 2px;
+      background-image: url('/img/arrow_up_right.svg');
```
Comment (agrim-deriv) [inline src/styles/index.scss:402]: Can we use quill-icons instead of using svg?

File: src/components/UserNavbarItem/item.desktop.tsx [hunk 2]
```diff
@@ -1,43 +1,94 @@
...
+  );
+};
 
 const UserNavbarDesktopItem = ({ authUrl, is_logged_in }: IUserNavbarItemProps) => {
-  const [toggle_search, setToggleSearch] = useState<boolean>(false);
+  const { logout } = useLogout();
+  const { deviceType } = useDeviceType();
+  const isDesktop = deviceType === 'desktop';
+
+  console.log('is_logged_in', is_logged_in);
```
Comment (agrim-deriv) [inline src/components/UserNavbarItem/item.desktop.tsx:82]: Was this meant for debugging, or we need this?

File: src/styles/index.scss (hunk 1 above)
Comment (ahmed-deriv) [reply src/styles/index.scss:402]: I didn't think so. In style file we can't import icon from there. I downloaded this icon from quill to use here.

File: src/components/UserNavbarItem/item.desktop.tsx (hunk 2 above)
Comment (ahmed-deriv) [reply src/components/UserNavbarItem/item.desktop.tsx]: Fixed

**Review by agrim-deriv — APPROVED** [review]

**Review by sandeep-deriv — APPROVED** [review]

**Pull request merged**